.github/workflows/sync-secrets.yml

```diff
@@ -0,0 +1,32 @@
+name: Sync Secrets
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: 'Dry Run'
+        required: true
+        default: 'true'
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+
+jobs:
+  sync-secrets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Sync Secrets
+        uses: jpoehnelt/secrets-sync-action@v1.9.0
+        with:
+          SECRETS: |
+            ^SECRET_TO_SYNC$
+          REPOSITORIES: |
+            ^link108/sync-secrets-to$
```

Comment:
Can we sync to https://github.com/spinnaker/spinnaker directly?

Reply:
ah yes, that was my intent, updated!